### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by utkarsh006**
# Pull Request Template

## Description

- See changes [here](https://github.com/utkarsh006/Big-Omega-Extension/tree/readme_update)

## PR checklist
- [x] Added all the relevant summary for the change
- [x] Updated README (if required)
- [x] These changes are backward compatible
- [x] Added snapshots.
- [x] Tested locally before publishing the PR

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Test A
- [x] Test B

## Make sure all below guidelines are followed else PR will get Reject:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code so that it is easy to understand
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by utkarsh006** [comment]
@GouravRusiya30 Kindly Review.

**Comment by GouravRusiya30** [comment]
Why is this Change for ?

**Comment by utkarsh006** [comment]
@GouravRusiya30 The motivation is simple. To format the text written in Readme in such a way that it should look concise and the user may not feel abrupt while reading the text. Shorter the text, the more attentio ,user can pay to it.

File: README.md [hunk 1]
```diff
@@ -16,91 +16,33 @@
...
 <h2>Find us on Telegram, Chrome-Webstore & Mozilla Firefox</h2>
 
 <a href="https://t.me/codecks"><img align="left" height=200 src="https://media.giphy.com/media/wlR4kWTnwEyY8RwHKM/giphy.gif"></a>
 [![Chrome](https://user-images.githubusercontent.com/53124886/111952712-34f12300-8aee-11eb-9fdd-ad579a1eb235.png)](https://chrome.google.com/webstore/detail/big-%CF%89-enhance-leetcoding/hfjfkofmpdgbfpkgnknikojbpljnkfkf) [![Firefox](https://user-images.githubusercontent.com/53124886/126341427-4a4e57aa-767a-467e-83d2-b31fa3564441.png)](https://addons.mozilla.org/en-US/firefox/addon/big-omega-extension/)
 
-## Big Ω - Demo (YouTube Video):
-<a href="http://www.youtube.com/watch?feature=player_embedded&v=IhRoe7p1V-I
-" target="_blank"><img src="https://github.com/codedecks-in/Big-Omega-Extension/blob/main/public/big-omega.png" 
-alt="Big Ω - Enhance Leetcoding experience" height="500" border="10" /></a>
+- Big Ω [Installation Video](https://youtu.be/IhRoe7p1V-I)
```
Comment (GouravRusiya30) [inline README.md:26]: Please don't change this, revert it to original (We want it to be displayed as an img)

Comment (utkarsh006) [reply README.md:26]: Sure

**Comment by utkarsh006** [comment]
@GouravRusiya30 Review now.

File: README.md [hunk 2]
```diff
@@ -16,91 +16,33 @@
...
-1. You will be able to see all the people in the world who are solving problems on various platforms
-2. Irrespective of platforms, you will be able have audio/chat conversation with online peers
-3. You will be able to create groups, start coding competitions by putting timer of 30m 45m or any custom time, while tracking each others progress about who solved how many test cases, etc
-4. You will also be able to sync each others screen so that you can explain logic in groups or in person with a first person view
+With this tool **as of now** we have launched a feature that reveals information about which companies have asked the problem that you are currently solving along with its frequency being asked.
 
-Please let us know about our plugin at our email - manage.bigomega@gmail.com
-your feedback and constructive criticism, is always appreciated, and we look forward to make this browser plugin better day by day, step by step
-  
-### BONUS: Star this [repository](https://github.com/codedecks-in/Big-Omega-Extension/) for further development of features. If you want a particular feature, simply request for it!
```
Comment (GouravRusiya30) [inline README.md:58]: Also, please keep this part as it is. We don't want to make this change.

Comment (utkarsh006) [reply README.md:58]: But I have made it shorter in text which would eventually help users only.

Comment (utkarsh006) [reply README.md:58]: @GouravRusiya30 It is looking relevant too.

File: README.md [hunk 3]
```diff
@@ -16,91 +16,38 @@
...
-  <a href="https://github.com/codedecks-in/Big-Omega-Extension/graphs/contributors" alt="Contributors">
-    <img src="https://img.shields.io/github/contributors/codedecks-in/Big-Omega-Extension" />
-  </a>
-  <a href="https://github.com/codedecks-in/Big-Omega-Extension/issues" alt="Contributors">
-    <img src="https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat" />
-  </a>
-  <a href="https://code.publiclab.org#r=all" alt="Contributors">
-    <img src="http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square" />
-  </a>
 </p>
```
Comment (GouravRusiya30) [inline README.md:19]: Why this is changed?

File: README.md (hunk 2 above)
Comment (GouravRusiya30) [reply README.md:58]: I cannot see that you have made it shorter. I can only see that it is removed.

**Pull request closed without merging**